.github/CODEOWNERS

```diff
@@ -1,29 +1,35 @@
 # Automatically request reviews when a pull request changes any owned files
-# More information: https://github.com/blog/2392-introducing-code-owners 
 
-*.yml @dotnet/roslyn-infrastructure
+# More information: https://github.com/blog/2392-introducing-code-owners
+
+\*.yml @dotnet/roslyn-infrastructure
```

Comment:
Why this change? GitHub says that's an error:

![image](https://github.com/user-attachments/assets/2064d0ca-6558-42ef-b632-981b9d326da7)

Reply:
So weird. I must have an extension that made these changes while formatting on save. I will revert.

Reply:
VSCode was treating the file as markdown.